vegafusion-python/vegafusion/dataset.py

```
@@ -0,0 +1,91 @@
+
+import uuid
+import weakref
+from typing import Any, ClassVar
+
+from arro3.core import Schema
+
+
+class _DataRef:
+    """Weak-referenceable wrapper holding an opaque data object."""
+
+    __slots__ = ("data", "__weakref__")
```

Ruff (0.15.6)

[warning] `_DataRef.__slots__` is not sorted

Apply a natural sort to `_DataRef.__slots__`

(RUF023)